src/hooks/useExperienceBuilder.ts

```diff
@@ -21,44 +19,41 @@ type UseExperienceBuilderProps = {
   mode?: ExternalSDKMode
 }
 
+/**
+ * @deprecated This hook is deprecated. We re-designed the SDK to enable client or server side rendering
+ */
```

Comment:
I know this is colliding with the suggestion made by @DanweDE but I would still keep this in the long run. Do we imagine every customer to have an advanced use case or why don't we keep the API easy to use for easy use cases?

Reply:
+1. For customers who don't wish to go through the process of pre-fetching the entities and creating an experience to get the experience object, how about we leave this easy setup hook?

Reply:
Not sure I understand, but there is [this new useFetchExperience hook](https://github.com/contentful/experience-builder/blob/feat/support-ssr/src/hooks/useFetchExperience.ts#L22) that we also export, which will be the substitute to the `useExperienceBuilder` hook.

Supporting `useExperienceBuilder` further makes it harder to separate functionality. For example, it required `slug` to be provided to the `ExperienceRoot`. Now that we have added the `id`, should now `ExperienceRoot` also require an `id` to be passed?

Deprecating it allows us to streamline the usage in the next major version as regardless whether used with SSR or SSR or CSR, in all 3 cases entities are going to be fetched before using the `ExpereinceRoot` and we will free our sdk from unreliable checks and assumptions to call the right fetch method - by id or by slug.